### DISCUSSION TITLE
504 Gateway time-out error

### DISCUSSION THREAD

**Discussion by Lucatik** [lllyasviel/Fooocus#2184] [Q&A] [answered]
Hi, I am experiencing some problems when using Fooocus in Colab. When I am clicking on; Running on public URL: https://blablabla.gradio.live/ I get an error. [504 Gateway time-out error]. It always worked fine for me until now. Anyone having the same problems? And does someone has the solution? 

Thank you so much!

**Comment by bou510** [comment]
I got same error!

**Reply by KDasanayaka** [reply]
running done
![Screenshot 2024-02-06 224932](https://github.com/lllyasviel/Fooocus/assets/153555278/ffca0aa6-256c-4e21-bcc3-74b3924d0c85)

**Comment by Abbat56** [comment]
+1 I'm seeing the same problem

**Reply by bou510** [reply]
yeah same problem here

**Comment by harshaldesign** [comment]
I also got same

**Comment by Lucatik** [comment]
Okay, so there are more with this problem. Let's see if somebody has a solution :)

**Comment by dmitrysinev** [comment]
Same problem! Yesterday everything was working without any problems

**Comment by bhuisman1992** [comment]
Same here!

**Comment by he-is-talha** [comment]
I think there's some problem with gradio.live

https://status.gradio.app/793595809#logs

**Reply by KDasanayaka** [reply]
yes

**Comment by KDasanayaka** [comment]
same problem - 504 Gateway time-out error
![Screenshot 2024-02-06 201626](https://github.com/lllyasviel/Fooocus/assets/153555278/0fb7eff8-01a9-4226-ae9a-50d4fc143993)
 
gradio server is down please wait
https://status.gradio.app/793595809#logs

**Comment by leejake98** [comment]
Getting the same error!

**Comment by he-is-talha** [comment, marked as answer]
UPDATE: It's fixed now. You can use it again normally :)

**Comment by Al-Rien** [comment]
The same problem again. Please, fix it.

**Reply by Ckrotky** [reply]
Fooocus has nothing to do with it. The problems are happening in Gradio. You can check the status at https://status.gradio.app/.
We can only wait for Gradio to be fixed...)
